### ISSUE TITLE
Smells a bit 🐟y

### ISSUE THREAD

**Issue by code-grey** [laptopsregisgov/apk#1]
Is it a zero day?
Ransom or something else, like spyware?
Got it through an SMS🤓
Haven't installed it yet though

**Comment by M3ruem** [comment]
It looks like a backdoor trojen my friend.

**Comment by M3ruem** [comment]
Nope it's not a backdoor! And why the hell did he have to encrypt this shit in AES and honestly he is the worst developer i've ever seen?
`Jio 5G-Sim Registration
Jio 5G is Launched with 10GB
data daily. Enter your mobile
number on Jio5G app and
register your Sim.`

**Comment by M3ruem** [comment]
Nope it's not a backdoor! And why the hell did he have to encrypt this shit in AES?
`Jio 5G-Sim Registration Jio 5G is Launched with 10GB data daily. Enter your mobile number on Jio5G app and register your Sim.`
 honestly he is the worst developer i've ever seen.

**Comment by code-grey** [comment]
> It looks like a backdoor trojan my friend.
Did you check it out yourself @M3ruem ?
Thanks a lot😄

**Comment by code-grey** [comment]
> Nope it's not a backdoor! And why the hell did he have to encrypt this shit in AES?
> `Jio 5G-Sim Registration Jio 5G is Launched with 10GB data daily. Enter your mobile number on Jio5G app and register your Sim.`
>  honestly he is the worst developer i've ever seen.
> 

Did you check it out yourself @M3ruem ? Thanks for the info!😄

**Comment by M3ruem** [comment]
yeah and i am still analyzing that apk's source code! But seriously i really had a lot of expectations and he seems like a biggest noobie guy! i mean what is even the point of making a malware when you don't even include a revershell in it?

**Comment by M3ruem** [comment]
hey @code-grey but still it's a spamware! so don't install it. It will send the messages and i also seen that he implemeted some sorta isp info grasper!  the app won't access your personal data but still it'll do a shitty spam! so beware of it

**Comment by code-grey** [comment]
> hey @code-grey but still it's a spamware! so don't install it. It will send the messages and i also seen that he implemeted some sorta isp info grasper!  the app won't access your personal data but still it'll do a shitty spam! so beware of it
> 

@M3ruem thanks a lot buddy, actually I'm more of a newbie in this field, so thanks for letting me know this ☺️
BTW if you don't mind can you upload the screenshots of the app? Moreover what did you use to decompile the apk? (I know I sound like a total noob but still, gotta start somewhere, right? 😉)

Thanks in advanced/You already helped me a lot, thanks!
`😆`
